### ISSUE TITLE
Smooth decrease in L1 Loss

### ISSUE THREAD

**Issue by shenoynikhil** [atomicarchitects/equiformer#4]
I noticed that the loss decreases smoothly for the qm9 logs. Any practical tips (like EMA) on how to achieve that?

**Comment by yilunliao** [comment]
Hi @shenoynikhil .

Thanks for the question.

The training loss in the logs is the loss averaged over all iterations within an epoch (see [here](https://github.com/atomicarchitects/equiformer/blob/a4360ada2d213ba7b4d884335d3dc54a92b7a371/engine.py#L85-L103)).
Since we average the loss, it can be somewhat smooth.

For validation loss, using EMA can make it smoother. 

I do not fully understand your question. 
You can find some training techniques in the code and paper.

Let me know if there is something specific you want to know.

**Comment by shenoynikhil** [comment]
The EMA makes sense. I was actually looking at L1 loss values per step without EMA which was quite jittery.
